salt/ext/saslprep.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2016-present MongoDB, Inc.
```

Comment:
I would like to avoid maintaining upstream code if possible. Is there a library we could be importing and using that instead?

Reply:
There is none that I could find. I discussed with multiple core salt devs the best way to do it.

Requiring users to install the Mongo driver in order to use PostgreSQL would not be good.

Reply:
K, just wanted to make sure there was not another solution here.